docs/pages/product/configuration/data-sources/databricks-jdbc.mdx

```diff
@@ -147,6 +165,17 @@ CUBEJS_DB_EXPORT_BUCKET_AZURE_CLIENT_ID=<AZURE_CLIENT_ID>
 CUBEJS_DB_EXPORT_BUCKET_AZURE_CLIENT_SECRET=<AZURE_CLIENT_SECRET>
 ```
 
+#### Google Cloud Storage
```

Comment:
There's two separate subsections in `### Export Bucket`, both named `#### Google Cloud Storage`

Reply:
Good catch! Thnx!